src/parser/parsers/attributes.py

```diff
@@ -138,42 +115,29 @@ def _parse_shop_stat_display(self, hero_value):
             "Weapon": [
                 "BulletDamage",
                 "LightMeleeDamage",
-                "WeaponAttribute_RapidFire",
-                "WeaponAttribute_MediumRange"
                 ]
         """
 
         category_attributes = {}
 
-        shop_stats_ui = hero_value['m_ShopStatDisplay']
-        for category, category_stats in shop_stats_ui.items():
-            category_name = maps.get_shop_attr_group(category)
-
-            # Ensure category exists
-            if category_name not in category_attributes:
-                category_attributes[category_name] = {}
-
-            # Process all stats in the category
-            for _, stats in category_stats.items():
-                if type(stats) is str:
-                    # Contains weapon type and weapon range
-                    # May be parsed in the future, left out of this data for now
-                    # stats = stats.split(' | ')
-                    continue
-                elif type(stats) is list:
-                    pass
-                else:
-                    raise Exception(f'Expected string or list, got {type(stats)}')
+        # Process all stats in the category
+        stats = []
+        if 'm_vecDisplayStats' in category_stats:
+            stats += category_stats['m_vecDisplayStats']
+        if 'm_vecOtherDisplayStats' in category_stats:
+            stats += category_stats['m_vecOtherDisplayStats']
 
-                # Add to parsed stats
-                for stat in stats:
-                    stat_mapped = maps.get_hero_attr(stat)
+        # Add to parsed stats
+        for stat in stats:
+            stat_mapped = maps.get_hero_attr(stat)
 
-                    # Add stat to category if not already present
-                    if stat_mapped not in category_attributes[category_name]:
-                        category_attributes[category_name][stat_mapped] = {}
+            # Add stat if not already present
+            if stat_mapped not in category_attributes:
+                category_attributes[stat_mapped] = {}
 
         return category_attributes
+
+
 """
 Output-data is used by https://deadlocked.wiki/Template:StatBoxes
 to display a hero's attributes on their hero page
```

Comment:
Has this always just been chillin at the bottom of the page?

Reply:
yeah, for a few PR's at least

Reply:
If its still relevant, can it not be on the class documentation?